packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -34,15 +36,31 @@ class ScaleStartDetails {
   /// Creates details for [GestureScaleStartCallback].
   ///
   /// The [focalPoint] argument must not be null.
-  ScaleStartDetails({ this.focalPoint = Offset.zero })
-    : assert(focalPoint != null);
+  ScaleStartDetails({ this.focalPoint = Offset.zero, Offset localFocalPoint, })
+    : assert(focalPoint != null), localFocalPoint = localFocalPoint ?? focalPoint;
 
   /// The initial focal point of the pointers in contact with the screen.
+  ///
   /// Reported in global coordinates.
+  ///
+  /// See also:
+  ///
+  ///  * [localFocalPoint], which is the same value reported in global
```

Comment:
"local" (and on 93)

Sorry for the drive-by review, I was just looking through the commit to learn more and spotted this easy-to-miss slip.

Reply:
+1